coops-ui/src/views/AnnualReport.vue

```diff
@@ -513,15 +513,11 @@ export default {
                 this.$refs.directorsList.setDraftDate(annualReport.annualGeneralMeetingDate)
               }
               if (this.entityFilter(EntityTypes.COOP)) {
-                // set the new AGM date in the AGM Date component
-                // (this value is empty if user did not enter an AGM date)
-                this.newAgmDate = annualReport.annualGeneralMeetingDate
-                // set the new No AGM flag in the AGM Date component
-                this.newNoAgm = annualReport.didNotHoldAgm
+                // set the new AGM date in the AGM Date component (may be null or empty)
+                this.newAgmDate = annualReport.annualGeneralMeetingDate || ''
```

Comment:
should this be empty string or null?

Reply:
Either one is fine. The AGMDate component takes this value in as a prop as follows:
```
  @Prop({ default: null })
  private newAgmDate: string | null
```
After that it's treated as truthy or falsy, where "" is falsy.